packages/react/src/utils.ts

```diff
@@ -538,7 +539,7 @@ export type ColumnValueType =
 
 export const isRoleAssignmentsArray = (value: ColumnValueType): value is RoleAssignmentsValueType[] => {
   if (!Array.isArray(value) || value.length === 0) return false;
-  if (!value.every((item) => typeof item === "object" && "__typename" in item && item.__typename === "Role")) return false;
+  if (!value.every((item) => item !== null && typeof item === "object" && "__typename" in item && item.__typename === "Role")) return false;
```

Comment:
If there's a null in the `value` array and `isRoleAssignmentsArray` returns false, we will go to an else block where we do 
```js
 formattedTags = value.map((tag) => tag.toString());
```

Over there, I think we need some null safety

Reply:
Added + a test for insurance. 👍 good catch.